docs/_breadcrumb/toc.yml

```diff
@@ -18,18 +18,9 @@
     - name: Consume packages
       tocHref: /nuget/consume-packages/
       topicHref: /nuget/consume-packages/overview-and-workflow
-    - name: Hosting packages
-      tocHref: /nuget/hosting-packages/
-      topicHref: /nuget/hosting-packages/overview
-    - name: Tools
-      tocHref: /nuget/tools/
-      topicHref: /nuget/tools/nuget-exe-cli-reference
     - name: Reference
       tocHref: /nuget/reference/
       topicHref: /nuget/reference/nuspec
-    - name: API
```

Comment:
Correct me if I'm wrong, but with your changes you now have to click on the "NuGet.org" documentation link to see the "NuGet API" reference. Considering the API isn't specific to NuGet.org, could we move this link to be under NuGet's `Reference` section?

Reply:
@loic-sharma  we don't really have docs on the client APIs. Everything I see today seems to fall under nuget.org APIs.

Reply:
>  Everything I see today seems to fall under nuget.org APIs.

I think you missed this line from my comment:

>  ... the API isn't specific to NuGet.org

We want people to migrate their tools from NuGet's V2 APIs to V3 APIs. To promote this migration, we need the V3 APIs' documentation to be easily discoverable.

> we don't really have docs on the client APIs.

We have basic docs for the client's code base: [NuGet Client SDK](https://docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk)

Reply:
It is easy to include a link to these ref docs in both TOC views for highest discoverability. If it is good to "prefer" one TOC view for these ref docs (say, under main NuGet Ref section), that is the easiest way to do it. There are other ways, too.

Reply:
I think linking to these docs in both TOC views would be great!

Reply:
Sound good...lets go with Loic's recommendation.